bundles/automation/org.eclipse.smarthome.automation.integration.test/org.eclipse.smarthome.automation.integration.test.launch

```diff
@@ -26,7 +26,7 @@
 <stringAttribute key="org.eclipse.jdt.junit.TESTNAME" value=""/>
 <stringAttribute key="org.eclipse.jdt.junit.TEST_KIND" value="org.eclipse.jdt.junit.loader.junit4"/>
 <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_START_ON_FIRST_THREAD" value="true"/>
-<stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+<stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```

Comment:
> I've changed required jvm because the test can be started. It depends from bundles which required 1,8 jvm and there are a lot of such exceptions:

Okay, all fine.

> When the ESH goes to jdk 1.8 we have decided with Kai that they can be 1.8 compact 2 (compact 2 because the GSON requires compact2). 

I don't get the point how does this fit to that comment:

> The idea to use Java8 Predicates seems out of scope as parts of ESH must be available in Java 1.5 environments.

https://github.com/eclipse/smarthome/issues/3281#issue-223322632